Intersect.Client/Interface/Game/DescriptionWindows/ItemDescriptionWindow.cs

```diff
@@ -0,0 +1,478 @@
+        {
+            // Create our header, but do not load our layout yet since we're adding components manually.
+            mHeader = AddHeader();
+
+            // Set up the icon, if we can load it.
+            var tex = Globals.ContentManager.GetTexture(Framework.Content.TextureType.Item, mItem.Icon);
+            if (tex != null)
+            {
+                mHeader.SetIcon(tex, mItem.Color);
+            }
+
+            // Set up the header as the item name.
+            CustomColors.Items.Rarities.TryGetValue(mItem.Rarity, out var rarityColor);
+            var name = !string.IsNullOrWhiteSpace(mTitleOverride) ? mTitleOverride : mItem.Name;
+            mHeader.SetTitle(name, rarityColor ?? Color.White);
+
+            // Set up the description telling us what type of item this is.
+            // if equipment, also list what kind.
+            Strings.ItemDescription.ItemTypes.TryGetValue((int) mItem.ItemType, out var typeDesc);
+            if (mItem.ItemType == ItemTypes.Equipment)
+            {
+                var equipSlot = Options.Equipment.Slots[mItem.EquipmentSlot];
+                var extraInfo = equipSlot;
+                if (mItem.EquipmentSlot == Options.WeaponIndex && mItem.TwoHanded)
+                {
+                    extraInfo = $"{Strings.ItemDescription.TwoHand} {equipSlot}";
+                }
+                mHeader.SetSubtitle($"{typeDesc} - {extraInfo}", Color.White);
+            }
+            else
+            {
+                mHeader.SetSubtitle(typeDesc, Color.White);
+            }
+
+            // Set up the item rarity label.
+            Strings.ItemDescription.Rarity.TryGetValue(mItem.Rarity, out var rarityDesc);
+            mHeader.SetDescription(rarityDesc, rarityColor ?? Color.White);
+
+            mHeader.SizeToChildren(true, false);
+        }
+
+        protected void SetupItemLimits()
+        {
+            // Gather up what limitations apply to this item.
+            var limits = new List<string>();
+            if (!mItem.CanBank)
+            {
+                limits.Add(Strings.ItemDescription.Banked);
+            }
+            if (!mItem.CanGuildBank)
+            {
+                limits.Add(Strings.ItemDescription.GuildBanked);
+            }
+            if (!mItem.CanBag)
+            {
+                limits.Add(Strings.ItemDescription.Bagged);
+            }
+            if (!mItem.CanTrade)
+            {
+                limits.Add(Strings.ItemDescription.Traded);
+            }
+            if (!mItem.CanDrop)
+            {
+                limits.Add(Strings.ItemDescription.Dropped);
+            }
+            if (!mItem.CanSell)
+            {
+                limits.Add(Strings.ItemDescription.Sold);
+            }
+
+            // Do we have any limitations? If so, generate a display for it.
+            if (limits.Count > 0)
+            {
+                // Add a divider.
+                mDivider1 = AddDivider();
```

Comment:
I'm confused about what each Divider really does, maybe something like mDividerLimits, mDividerDescription, etc would be more suggestive